examples/README.md

```diff
@@ -1,4 +1,8 @@
 # JupyterLite Examples
 
 This folder contains example notebooks and files demoing what is possible to do with
-JupyterLite.
+JupyterLite. To get started, there is a short introduction in [the `intro.ipynb` notebook](intro.ipynb).
```

Comment:
Pushing a commit to fix the lint failure.

Reply:
Done in https://github.com/jupyterlite/jupyterlite/pull/432/commits/6a054a618bbe59bccf89eb9964216e130e51dba0